pkgs/by-name/gk/gk-cli/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+}:
+
+let
+  version = "2.1.0";
+in
+stdenv.mkDerivation {
+  pname = "gk-cli";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/gitkraken/gk-cli/releases/download/v${version}/gk_${version}_Linux_x86_64.zip";
+    sha256 = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  postInstall = ''
+    mkdir --parents $out/bin/
+    install ./gk $out/bin/
+
+    installShellCompletion --bash ./**/gk.bash
+    installShellCompletion --fish ./**/gk.fish
+    installShellCompletion --zsh ./**/_gk
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/gk \
+      --prefix PATH : "${lib.makeBinPath [ coreutils ]}"
+  '';
+
+  meta = {
+    description = "GitKraken CLI is the ultimate CLI for Git collaboration";
+    homepage = "https://www.gitkraken.com";
+    license = lib.licenses.unfree;
+    maintainers = [ lib.maintainers.pinage404 ];
+    mainProgram = "gk";
+    platforms = [ "x86_64-linux" ];
```

Comment:
Upstream offers binaries for other architectures and operating systems, how about expanding this package, so that these are also supported?

Reply:
yes, I'm trying in another branch, I'm just about to make it!

Reply:
Is it ok to merge this one first, then do another PR to add the others platforms ?

Reply:
My preference is to add them to this PR, see comment on `src` above.
I also recommend to squash all commits into a single one prior final review, makes it a bit easier on the package and git history 🙂